distribution/docs/src/main/resources/_contents/introductioncontents.adoc

```diff
@@ -0,0 +1,44 @@
+${ddf-branding-expanded} ${branding} is a free and open-source common data layer that abstracts services and business logic from the underlying data structures to enable rapid integration of new data sources.
+
+Licensed under http://www.gnu.org/licenses/gpl.html[LGPL], ${ddf-branding} is an interoperability platform that provides secure and scalable discovery and retrieval from a wide array of disparate sources.
+
+${ddf-branding} is:
+
+* a flexible and modular integration framework.
+* built to "unzip and run" while having the ability to scale to large enterprise systems.
+* primarily focused on data integration, enabling clients to insert, query, and transform information from disparate data sources via the ${ddf-branding} Catalog.
+
+=== Applications
+
+${branding} is comprised of several modular applications, to be installed or uninstalled as needed.
+
+${ddf-branding} Administrative Application::
+The administrative application enhances administrative capabilities when installing and managing ${branding}. It contains various services and interfaces that allow administrators more control over their systems.
+
+${ddf-branding} Catalog Application::
+The ${branding} Catalog provides a framework for storing, searching, processing, and transforming information.
+Clients typically perform local and/or federated query, create, read, update, and delete (QCRUD) operations against the Catalog.
+At the core of the Catalog functionality is the *Catalog Framework*, which routes all requests and responses through the system, invoking additional processing per the system configuration.
+
+${ddf-branding} Content Application::
+The ${ddf-branding} Content application provides a framework for storing, reading, processing, transforming and cataloging data.
+
+${ddf-branding} Platform Application::
+The Platform application is considered to be a core application of the distribution. The Platform application has fundamental building blocks that the distribution needs to run. +
+These building blocks include subsets of: http://karaf.apache.org/[Karaf], http://cxf.apache.org/[CXF], and http://camel.apache.org/[Camel]. +
```

Comment:
For some reason I can't figure out, it looks like asciidoc is not rendering the Camel link as [Camel](http://camel.apache.org/) but rather as "http://camel.apache.org/[Camel]"